nnpdf_data/nnpdf_data/athena_utils.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+from pathlib import Path
+from typing import Optional, Union
+
+import yaml
+import numpy as np
+
+
+def read_excel(path_xlsx: Path, beams: tuple) -> pd.DataFrame:
+    """Parse the xlsx file containing all the information regarding
+    the projections and returns the ones corresponding to the chosen
+    beam energies.
+
+    Parameters
+    ----------
+    path_xlsx : Path
+        path to the xlsx file
+    beams: tuple
+        tuple specifying the beam energies of the lepton & proton
+
+    Returns
+    -------
+    pd.DataFrame
+        returns a panda table corresponding to the chosen beams
+    """
+    xdf = pd.read_excel(path_xlsx)
+    el, ep = beams
+    df_beam = xdf[(xdf["El"] == el) & (xdf["Eh"] == ep)]
+    return df_beam[df_beam["y"] <= 1]
```

Comment:
`df_beam["y"] <= 1` is always true so why put it here?

Reply:
This was not the case for the previous rawdata and I'd leave it there for safety given that these datasets might be subjected to modifications (depending on updated projections).

Reply:
If I remember correctly what was written in the ATHENA paper, this is an artefact of how the detector simulations are designed in the projections. What is happening is that when the detector simulations are combined with the MC, there are events (at large-$x$) in which the electrons receive larger energies rendering $y > 1$.  These are ofc unphysical events and therefore must be discarded. They nevertheless dump them in the rawdata and we have to remove them manually.

Reply:
Thanks for the explanation. I think it's also worth adding that in a comment here

Reply:
Addressed. I added some comments to reflect this.